src/overlays/actors/ovl_Obj_Boat/z_obj_boat.h

```diff
@@ -3,12 +3,23 @@
 
 #include "global.h"
 
+#define OBJBOAT_GET_PARAMS(thisx) ((thisx)->params)
+#define OBJBOAT_GET_PATH(thisx) (((thisx)->params >> 0x07) & 0x1F)
+#define OBJBOAT_GET_4000(thisx) ((thisx)->params & 0x4000)
+
 struct ObjBoat;
 
 typedef struct ObjBoat {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x24];
-} ObjBoat; // size = 0x168
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ u8 posX;
+    /* 0x15D */ s8 rotY;
```

Comment:
It's not obvious to me that either of these names is right, have you got a reason to name these like this?